planner_cspace/src/planner_3d.cpp

```diff
@@ -1350,6 +1347,7 @@ class Planner3dNode
     as_.setSearchTaskNum(num_task);
     pnh_.param("num_cost_estim_task", num_cost_estim_task_, num_threads * 16);
 
+    pnh_.param("retain_last_error_status", retain_last_error_status_, false);
```

Comment:
default value is better to be compatible with the previous version.

Reply:
addressed